src/openrct2/ride/Vehicle.cpp

```diff
@@ -1814,7 +1814,7 @@ static void vehicle_update_measurements(rct_vehicle* vehicle)
 
     TileElement* tile_element = map_get_surface_element_at({ x, y });
     // If vehicle above ground.
-    if (tile_element->base_height * 8 <= z)
+    if (tile_element && tile_element->base_height * 8 <= z)
```

Comment:
We normally write ` != nullptr`.
Also, could you amend the changelog?

Reply:
Sure